test/storage/tile_group_test.cpp

```diff
@@ -232,11 +238,14 @@ TEST_F(TileGroupTests, StressTest) {
 //  std::vector<catalog::Schema> schemas;
 //
 //  // SCHEMA
-//  catalog::Column column1(type::TypeId::INTEGER, type::Type::GetTypeSize(type::TypeId::INTEGER),
+//  catalog::Column column1(type::TypeId::INTEGER,
```

Comment:
I understand these are just formatting changes... but wrt to the MVCCInsertTest.
It is entirely commented out (has been for a while). Is there a reason why we are keeping it but disabling it?
Unless there is a good reason, I would delete the test.